### PR TITLE
Add api version to context

### PR DESCRIPTION
**Pull request by yongman**
Signed-off-by: yongman <yming0221@gmail.com>

### CODE REVIEW

File: include/pingcap/kv/Cluster.h [hunk 1]
```diff
@@ -36,7 +38,14 @@ struct Cluster
         , rpc_client(std::make_unique<RpcClient>(config))
         , oracle(std::make_unique<pd::Oracle>(pd_client, std::chrono::milliseconds(oracle_update_interval)))
         , lock_resolver(std::make_unique<LockResolver>(this))
-    {}
+    {
+        if (config.api_version == "v1")
```
Comment (sunxiaoguang) [inline include/pingcap/kv/Cluster.h:42]: Did we convert config to lower case before compare?

**Comment by sunxiaoguang** [comment]
I didn't find changes related to keyspace prefix. Is it missing?

File: include/pingcap/kv/Cluster.h (hunk 1 above)
Comment (yongman) [reply include/pingcap/kv/Cluster.h:42]: Yes, the caller must be set the api version config in lowwer case.

**Comment by yongman** [comment]
@sunxiaoguang The keyspace prefix related changes already merged in https://github.com/tikv/client-c/pull/105 and https://github.com/tikv/client-c/pull/106 .

**Pull request merged**